packages/webapp-alternative/src/components/FunctionalComponent.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react'
+
+const FunctionalComponent = () => {
+  const [count, setCount] = useState(0);
+
+  //increase counter
+  const increase = () => {
+    setCount(count => count + 1);
+  };
+
+  //decrease counter
+  const decrease = () => {
```

Comment:
As I mentioned in the chat. Let's follow the React naming convention for handlers. Link to the section https://reactjs.org/tutorial/tutorial.html#lifting-state-up
<img width="727" alt="Screen Shot 2022-05-12 at 17 57 58" src="https://user-images.githubusercontent.com/24767181/171344471-93d7de28-c0f0-458f-8d14-d96114c0c05d.png">